pkgs/development/compilers/flutter/package-overrides/default.nix

```diff
@@ -0,0 +1,4 @@
+{ callPackage }:
+
+{
```

Comment:
is this going to be used? is it just intentionally left empty as we do not need any patches right now?

Reply:
It [will be used](https://github.com/NixOS/nixpkgs/pull/201027#issuecomment-1510437924), but that can come in another PR.